src/Files.App/Actions/Open/OpenInVSAction.cs

```diff
@@ -31,11 +31,7 @@ public OpenInVSAction()
 		}
 
 		public Task ExecuteAsync()
-		{
-			Win32API.RunPowershellCommand($"start \'{_context.SolutionFilePath}\'", false);
-
-			return Task.CompletedTask;
-		}
+			=> Win32API.RunPowershellCommandAsync($"start \'{_context.SolutionFilePath}\'", false);
```

Comment:
@d2dyno1 do we have a consensus on how this should be formated?

Reply:
I think not using '=>' would be better

Reply:
We have to use braces in the all method implementations, according to. I was changing those all unupdated code to use braces. It enables us to add more lines easily without any unrelated changes.

Reply:
@0x5bfa can you file an issue/open a PR on the website repo?

Reply:
Sure

Reply:
I added the braces back

Reply:
Thanks!